src/library_pthread.js

```diff
@@ -242,7 +242,9 @@ var LibraryPThread = {
     // onFinishedLoading: A callback function that will be called once all of
```

Comment:
Existing issue, comment on line 241 looks wrong, it mentions a list of workers. Maybe correct that while you're editing the code near it?

Reply:
Done.